spotless/eclipse-java-formatter.xml

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="16">
+    <profile kind="CodeFormatterProfile" name="Spotless" version="16">
```

Comment:
This may not be formatting, but do we have any thoughts on using `this.` when it's not needed? Also, I apologize if some of this stuff is in the xml and I just missed it